docs/_includes/Grammar.md

```diff
@@ -177,10 +186,24 @@ Characters that can appear in a verbatim string.
 ### Comments
 Comments are in two forms.
 
-* They may go from "/*" to "*/" and be nested.
+* They may go from "/\*" to "\*/" and be nested.
 * They may go from "//" to the end of the line.
 
-## Tokens
+Note that the nesting of multi-line comments is behavior that is different 
+from most programming languages. In dafny,
```

Comment:
I didn't realize they are uncommon? They are supported in Pascal, Modula-2, Modula-3, Oberon, Scala, Haskell, and Boogie, for example.